lib/navigator.js

```diff
@@ -32,7 +32,10 @@ module.exports = function createNavigator(channel, currentExtensionId) {
       return channel.call('navigateToContentEntity', { ...opts, entityType: 'Asset', id: null })
     },
     openPageExtension: opts => {
-      return channel.call('navigateToPageExtension', { id: currentExtensionId, ...opts })
+      return channel.call('navigateToPage', { type: 'extension', id: ids.extension, ...opts })
+    },
+    openCurrentAppPage: opts => {
+      return channel.call('navigateToPage', { type: 'app', id: ids.app, ...opts })
```

Comment:
I'd put `id` after `opts` so it's not possible to override

Reply:
I did this purposely so it is possible to override. Right now the web app won't accept the override but in the future we may want to accept certain overrides.